layouts/New/layouts.module.css

```diff
@@ -142,26 +142,15 @@
   @apply flex
     w-full
     justify-center
+    gap-4
     bg-gradient-subtle
+    px-4
+    py-12
     dark:bg-gradient-subtle-dark
+    md:px-14
+    lg:px-28
     xs:bg-none
     xs:dark:bg-none;
-
-  main {
```

Comment:
As I asked before, and I'll ask again, please revert these unrelated changes.

Reply:
ThereÄs a reason why this ws done this way, and if there's no real reason behind this change, undo it. And make another PR with this change were we can discuss about it.